<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionsClass</class>
 <widget class="QWidget" name="QuestionsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Questions</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Questions:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="questionslistWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Enter a question:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="textlineEdit"/>
       </widget>
       <widget class="QPushButton" name="addpushButton">
        <property name="text">
         <string>Add Question</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Answers:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="answerslistWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Add answer:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="answerlineEdit"/>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="answerpushButton">
       <property name="text">
        <string>Add Answer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Questions.qrc"/>
 </resources>
 <connections/>
</ui>
